web/packages/teleport/src/services/history/history.ts

```diff
@@ -53,15 +53,22 @@ const history = {
     window.location.reload();
   },
 
-  goToLogin(rememberLocation = false) {
+  goToLogin(rememberLocation = false, withAccessChangedMessage = false) {
```

Comment:
These arguments should be replaced with an object, `websession.logoutWithoutSlo(true, false)` aren't very readable :(